File: chatwoot/kotlin/src/main/kotlin/oseg/chatwoot_examples/ConversationAddLabelsExample.kt
package oseg.chatwoot_examples

import com.chatwoot.client.infrastructure.*
import com.chatwoot.client.apis.*
import com.chatwoot.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class ConversationAddLabelsExample
{
    fun conversationAddLabels()
    {
        ApiClient.apiKey["api_access_token"] = "USER_API_KEY"
        // ApiClient.apiKey["api_access_token"] = "AGENT_BOT_API_KEY"
        // ApiClient.apiKey["api_access_token"] = "PLATFORM_APP_API_KEY"

        val conversationAddLabelsRequest = ConversationAddLabelsRequest(
            labels = listOf (),
        )

        try
        {
            val response = ConversationLabelsApi().conversationAddLabels(
                accountId = 0,
                conversationId = 0,
                _data = conversationAddLabelsRequest,
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling ConversationLabelsApi#conversationAddLabels")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling ConversationLabelsApi#conversationAddLabels")
            e.printStackTrace()
        }
    }
}

File: chatwoot/typescript-node/src/ConversationAddLabelsExample.ts
import * as fs from 'fs';
import api from "chatwoot_client"
import models from "chatwoot_client"

const apiCaller = new api.ConversationLabelsApi();
apiCaller.setApiKey(api.ConversationLabelsApiApiKeys.userApiKey, "USER_API_KEY");
// apiCaller.setApiKey(api.ConversationLabelsApiApiKeys.agentBotApiKey, "AGENT_BOT_API_KEY");
// apiCaller.setApiKey(api.ConversationLabelsApiApiKeys.platformAppApiKey, "PLATFORM_APP_API_KEY");

const contactAddLabelsRequest: models.ContactAddLabelsRequest = {
  labels: [
  ],
};

apiCaller.conversationAddLabels(
  0, // accountId
  0, // conversationId
  contactAddLabelsRequest, // data
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling ConversationLabelsApi#conversationAddLabels:");
  console.log(error.body);
});
